docs/extensions/tutorials/_posts/1970-01-08-ModifiyingExtensions.md

```diff
@@ -125,6 +125,23 @@ This is what we wanted! It was quite easy and straightforward. However, doing th
 
 To get those updates from Shoutem, we want to extend the extension instead of changing its code directly.
 
+### Debugging directly edited extensions
+
+When directly editing Shoutem extensions you will have to either re-clone the app after uploading and installing the extension into your app or re-naming the extension's directory name locally, otherwise you will experience issues on the server
```

Comment:
`otherwise you might experience issues on the Builder`. 
Might, because this will happen only if the uninstall the original extension.

Reply:
Add a "note". When forking extensions, it's best to re-clone the app afterwards to have everything in sync.